packages/asc-ui/src/components/Grid/Column.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Theme } from '@datapunt/asc-core'
+import ColumnStyle from './ColumnStyle'
+
+type Props = {
+  children: React.ReactNode
+  debug?: boolean
+  debugColor?: string
+  id?: keyof Theme.GridInterface
+  order?: Theme.TypeSpan
+  push?: Theme.TypeSpan
+  span: Theme.TypeSpan
+  styles?: any
```

Comment:
Rename to css to match other components?

Reply:
Hm, maybe I should remove that.